components/contact-details.tsx

```diff
@@ -38,9 +39,10 @@ const ReportButton = (props: ReportButtonProps) => (
     <button
       className="bg-white rounded-md font-medium text-indigo-600 hover:text-indigo-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
       onClick={props.onClick}
+      title="Laporkan kesalahan"
```

Comment:
```suggestion
      aria-label="Laporkan kesalahan"
```

I think `aria-label` is a slightly better practice over `title`.